pkg/namesgenerator/names-generator.go

```diff
@@ -569,6 +578,9 @@ var (
 
 		// Ada Yonath - an Israeli crystallographer, the first woman from the Middle East to win a Nobel prize in the sciences. https://en.wikipedia.org/wiki/Ada_Yonath
 		"yonath",
+
+		// Konrad Zuse - a German engineer, inventor of the first programmable computer. https://en.wikipedia.org/wiki/Konrad_Zuse
```

Comment:
Wondering if this would be controversial;

> Much of his early work was financed by his family and commerce, but after 1939 he was given resources by the Nazi German government

> While Zuse never became a member of the Nazi Party, he is not known to have expressed any doubts or qualms about working for the Nazi war effort.